root/statistics/Editors.js

```diff
@@ -0,0 +1,128 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {l_statistics as l} from '../static/scripts/common/i18n/statistics';
```

Comment:
I understand in this case we still need to keep the import for the aliasing?

Reply:
Correct.